services/cloudaccounts/aws/aws.go

```diff
@@ -91,6 +91,23 @@ type CloudAccountUpdateRegionConfigRequest struct {
 	} `json:"data,omitempty"`
 }
 
+type CloudAccountUpdateOrganizationalIdRequest struct {
```

Comment:
`CloudAccountUpdateOrganizationalIdRequest ` should be `CloudAccountUpdateOrganizationalIDRequest `